src/main/java/uk/gov/companieshouse/disqualifiedofficers/delta/serialization/ChsDeltaSerializer.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.companieshouse.disqualifiedofficers.delta.serialization;
+
+import org.apache.avro.io.DatumWriter;
+import org.apache.avro.io.EncoderFactory;
+import org.apache.avro.specific.SpecificDatumWriter;
+import org.apache.kafka.common.serialization.Serializer;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.delta.ChsDelta;
+import uk.gov.companieshouse.kafka.exceptions.SerializationException;
+import uk.gov.companieshouse.kafka.serialization.AvroSerializer;
+
+@Component
+public class ChsDeltaSerializer implements Serializer<ChsDelta> {
+
+    @Override
+    public byte[] serialize(String var1, ChsDelta chsDelta) {
```

Comment:
Should we not have unit tests for this class?

Reply:
Can do, I think insolvency logic for not unit testing was that it's called in the deserializer test and the consumer integration test so if it wasn't working properly those two tests would fail. Definitely doesn't hurt to add one in if we think it's worthwhile?